BOJ/solved.ac_class/Class03/1463.1로 만들기/sAp00n.py

```diff
@@ -0,0 +1,34 @@
+from sys import stdin
+from collections import deque
+
+
+def sol():
+    input_num = int(stdin.readline())
+    if input_num == 1:
+        print(0)
+        return
+    que = deque()
+    n = 0
+    que.append([input_num])
+
+    while True:
+        list_to_compute = que.pop()
+        # print(f'list to compute : {list_to_compute}')
+        result = []
+        temp = []
+        for ele in list_to_compute:
+            temp += [ele // 2 if ele % 2 == 0 else None, ele // 3 if ele % 3 == 0 else None, ele - 1]
+        for i in temp:
+            if i is None:
+                continue
+            result.append(i)
+        # print(f'temp : {temp}   result : {result}')
+        n += 1
+        if 1 in result:
+            print(n)
+            return
+        else:
+            que += [result]
```

Comment:
```suggestion
        if 1 in result:
            return print(n)
        que += [result]
```
어차피 if에서 return을 하기 때문에 else는 필요없을 것 같아요!

Reply:
아 이부분 수정한다는걸 뺴먹었네요 ㅋㅋ 리뷰 감사합니다!